### DISCUSSION TITLE
Debugger for zeus

### DISCUSSION THREAD

**Discussion by Souza-R** [Vdauphin/HeartsAndMinds#1132] [Question and answer]
Hello Guys.

Is there any way to use a debugger as Zeus ? or only for admins?

**Comment by Vdauphin** [comment]
Hello,
you can execute some of those lines in debug console
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/init_player.sqf#L42-L49
But not really userfriendly
Cheers
